R/dots.R

```diff
@@ -33,6 +33,23 @@
 #'
 #' @name dyn-dots
 #' @aliases tidy-dots
+#'
+#' @examples
+#' f <- function(...) {
+#'   out <- list2(...)
+#'   rev(out)
+#' }
+#'
+#' # splice
+#' x <- list(alpha = "first", omega = "last")
+#' f(!!!x)
+#'
+#' # unquote a name
+#' nm <- "key"
+#' f(!!nm := "value")
```

Comment:
Actually this should now probably be `"{nm}" := "value"`? Perhaps add an example with a constant prefix/suffix too?